opencog/attention/README

```diff
@@ -1,315 +1,7 @@
-hebbian links as the only diffusion mechanism. Without this property, 
-importance will not spread properly along non-hebbian atoms ocurring in 
-clauses, and will sometimes get "stuck" at certain atoms.
-
-This implementation is currently highly experimental. There are notes in the 
-implementation indicating additional functionality that is needed, including
-the ability to handle other types of hebbian links than AsymmetricHebbianLink. 
-
-There are also notes describing the functions that are a matter of ongoing 
-research. Particularly, the determination of what functions to use to determine
-the allocation of importance between source and target atoms, when the target
-atoms are a mix of hebbian and non-hebbian atoms, and the truth values of the
-target atoms have varying degrees of strength and confidence.
-
-The implementation requires benchmarking in order to determine its performance 
-profile. Additionally, further testing is currently underway with more complex
-graph structures.
-
-=== ForgettingAgent ===
-
-The ForgettingAgent, carries out the forgetting process in OpenCog Prime. It
-does based on the LTI of Atoms. Low LTI indicates that an atom has not been of
-any use for a long time, and additionally, isn't near any other important atoms.
-The latter condition is because the ImportanceSpreadingAgent would otherwise
-increase the STI of the atom, by moving STI from nearby important atoms, and
-increase the likelihood of the atom in question being:
-
-   1. used in mind processes, and thus
-   2. rewarded with stimulus which later gets exchanged for STI. 
-
-The ForgettingAgent also takes into account the VLTI of an atom. This is a
-boolean value that indicates whether the atom in fact is allowed to be
-forgotten. This allows a mechanism for ensuring that highly important
-information will never be forgotten, even if it's used only very very rarely.
-
-Forgetting can be tuned via two parameters:
-
-   1. the maximum LTI value that can be forgotten, and
-   2. the percentage of the AtomSpace to forget (typically this would be very low!) 
-
-These work in concert to limit how much and what atoms are forgotten. If only
-one parameter is set, then the other has free reign. I.e. a certain percentage
-of the AtomSpace will always be forgotten regardless of their LTI, or, any atom
-that drops below the maximum forgetting LTI will be forgotten.
-
-=== HebbianCreationModule ===
-
-The HebbianCreationModule creates AsymmetricHebbianLinks between atoms in the
-AttentionalFocus for updating by the HebbianUpdatingAgent. Rather than being a
-MindAgent that runs every cognitive cycle, it is implemented as a Module that is
-notified of atoms that are added to the AttentionalFocus by registering a slot
-with the Boost Signals2 events exposed by the AtomSpace. When this slot is
-called, a separate handler thread is created to do the work while allowing
-execution to continue.
```

Comment:
I just moved the old README into  old_agents folder. This README was just for msigana to have an easier time understanding it. A proper README is still on my todo list.